_topic_maps/_topic_map.yml

```diff
@@ -107,6 +107,30 @@ Topics:
   File: admission-plug-ins
   Distros: openshift-enterprise,openshift-aro,openshift-origin
 ---
+Name: Disconnected environments
+Dir: disconnected
+Distros: openshift-enterprise,openshift-origin
+Topics:
+- Name: Updating a cluster in a disconnected environment
+  Dir: updating
+  Topics:
+    - Name: About cluster updates in a disconnected environment
+      File: index
+    - Name: Mirroring OpenShift Container Platform images
+      File: mirroring-image-repository
+    - Name: Updating a cluster in a disconnected environment using OSUS
+      File: disconnected-update-osus
+      Distros: openshift-enterprise
+    - Name: Updating a cluster in a disconnected environment without OSUS
```

Comment:
```suggestion
    - Name: Updating a cluster in a disconnected environment without using OSUS
```

Reply:
Similar to the other comment, the name of the actual procedure is "Updating a cluster in a disconnected environment without the OpenShift Update Service".

Although I will keep these topic map titles as-is, this is definitely making me reconsider the name of these assemblies, so thanks!